multimetric-demo/finding_multiple_optima.ipynb

```diff
@@ -90,7 +90,7 @@
     "multimodal_first_solution_meta = {\n",
     "    'name': 'SigOpt Multimetric Demo - Single Metric Optimization (python)',\n",
     "    'project': 'sigopt-examples',\n",
-    "    'metrics': [{'name': 'value'}],\n",
+    "    'metrics': [{'name': 'value', 'objective': 'maximize'}],\n",
```

Comment:
hmm looks like we have used `value` in the past... seems like we should rename this

Reply:
I agree, going further, we can rename all of the `value` metrics to `score` instead?

Reply:
hmm I think in some cases there are actual metrics that are being optimized, like `accuracy`. I think it'd be best to infer from the example what the objective metric and use that? In some examples they just use a function evaluation which I think would be suitable to call `score`